events_lab.js

```diff
@@ -0,0 +1,73 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+
+    let form = document.querySelector("form")//no # needed because there's only 1 form & no id
+    let ul = document.getElementById("ulToDo")//no # b/c using getElbyId so it knows it's the tag
+    let input = document.querySelector("#newToDo")//needs # becise we're using tag name w/querySelector
+
+    form.addEventListener("submit", (e) => {
+        e.preventDefault();
+        let paragraph = document.createElement("p");
+        let emptyError = document.querySelector("p");
+
+        if(input.value === "") {
```

Comment:
In case the user has `"      "` having `input.value.trim()`  removes the empty spaces and returns `""` and then you can compare to an empty string.